transfromage/libs/api/Entities/player/Player.lua

```diff
@@ -46,11 +46,11 @@ Player.update = function(self, packet)
 	self.id = packet:read32() -- Temporary id
 
 	self.isShaman = packet:readBool()
-	self.isDead = packet:readBool()
+	self.isDead = packet:read8() > 0 -- Unsure what values above 1 mean?
 
 	self.score = packet:read16()
 
-	self.hasCheese = packet:readBool()
+	self.hasCheese = packet:read8() > 0 -- Cheeses
```

Comment:
Since it's 8b (up to 0xFF) then it can only be the number of cheese on the mouse ([0-3]).

Please add both fields

```Lua
self.cheese = packet:read8()
self.hasCheese = self.cheese > 0
```